care/emr/api/viewsets/user.py

```
@@ -192,3 +193,69 @@ def pnconfig(self, request, *args, **kwargs):
                 setattr(user, field, request.data[field])
         user.save()
         return Response({})
+
+    @action(detail=False, methods=["POST"])
+    def create_service_account(self, request, *args, **kwargs):
```

Ruff (0.14.10)

Unused method argument: `args`

(ARG002)